test/test-pexprs-madlibs.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+var test = require('tape');
+
+var makeGrammar = require('./testUtil').makeGrammar;
+
+// --------------------------------------------------------------------
+// Tests
+// --------------------------------------------------------------------
+
+// test('PExpr');
+//
+test('Alt', function(t) {
+  var g = makeGrammar([
+    'G {',
+    '  alt = "a" | "b"',
+    '}'
+  ]);
+
+  var random = Math.random;
```

Comment:
You should probably do this in a try/finally block, otherwise if a test throws an exception, you'll never reinstate the real `Math.random`, and it may cause weird errors in other tests.

I.e., make it:

```js
test('whatever', function(t) {
  var random = Math.random;
  try {
    /* test body */
  } finally {
    Math.random = random;
  }
  t.end();
}
```